internal/cloud/backend_runTasks.go

```diff
@@ -0,0 +1,144 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-tfe"
+)
+
+type taskResultSummary struct {
+	unreachable     bool
+	pending         int
+	failed          int
+	failedMandatory int
+	passed          int
+}
+
+type taskStageReadFunc func(b *Cloud, stopCtx context.Context) (*tfe.TaskStage, error)
+
+func summarizeTaskResults(taskResults []*tfe.TaskResult) *taskResultSummary {
+	var pe, er, erm, pa int
```

Comment:
Yes, I totally agree with this change. I guess I was trying to stick a bit to the "Go way" of naming variable, which usually is one letter or very few letters (like "p", "ret", "r", more on that topic: https://blog.devgenius.io/write-better-go-code-by-keeping-your-variable-names-short-13f404ac515c), which I actually find a bit annoying because, like you, I prefer descriptive names. Also, I have noticed that this project does not care much for that type of naming conventions, so I think is best for me to stay away from single letters :) @barrettclark

Reply:
It's wild to me that the Go way would favor terse over descriptive variables. When future me drops into old code and see something like `xzz` I have to spend more time trying to remember what the heck that means and why. I will try to not swim against the current if our style guide favors brevity, but very much appreciate more verbose and meaningful names in the meantime. Thank you for making this change.